profiles.yml

```diff
@@ -0,0 +1,6 @@
+duckdb:
```

Comment:
Is this required for adding the saved query? Or are you just adding it at the same time?

Reply:
This doesn't pertain to saved queries specifically, but I added it to get the codebase working locally in a devcontainer.